code/index.html

```diff
@@ -3,28 +3,184 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Take on Le Tour</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="icon" type="image/png" href="./images/cyclingIcon.png">
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@200&family=PT+Sans&display=swap" rel="stylesheet">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <header>
+      <div class="video-container">
+      <div id="hero">
+            <video
+            loop
+            muted
+            autoplay
+            preload="auto"
+            src="./videos/Video.mp4"
+            type="video.mp4"
+            type="video/webm">
+            <img src="./images/cyclist.jpg" alt="Front wheel and handlebar of a road cylist">
+        Your broswer does not support this video format.
+      </video>
+      <div class="caption">
+        <p>
+          We take you all the way
+          <span> 
+            It's your Tour! 
+          </span>
+          &mdash; for the road cycling enthusiast 
+          &mdash;
+        </p>
       </div>
+    </header>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <main>
+            
+        <section class="container-1">
+        <div class="form-wrapper">
+        <div class="forms">
+        <div class="login-form">
+            <input type="radio" name="tab" class="tab" id="sign-up-tab" checked />
+            <label for="sign-up-tab" class="tab-header">SIGN UP</label>
+            <input type="radio" name="tab" class="tab" id="sign-in-tab" />
+            <label for="sign-in-tab" class="tab-header">SIGN IN</label>
+            <form id="form1"
+            <form 
+              class="form"
+              action="http://httpbin.org/anything"
+              method="POST"
+              >
+              <div class="header">Sign up for free</div>
+              <div class="form-input">
```

Comment:
Please check your indentation, code structure is important and this is a bit hard to read